ClangSharpPInvokeGenerator/Properties/launchSettings.json

```diff
@@ -2,17 +2,11 @@
   "profiles": {
     "Regenerate ClangSharp - Windows": {
       "commandName": "Project",
-      "commandLineArgs": "--m clang --p clang_ --namespace ClangSharp --output ../../../../ClangSharp/Generated.cs --libraryPath $(LibClangName) --include $(LLVMIncludePath) --file $(LLVMIncludePath)/clang-c/Index.h --file $(LLVMIncludePath)/clang-c/CXString.h --file $(LLVMIncludePath)/clang-c/Documentation.h --file $(LLVMIncludePath)/clang-c/CXErrorCode.h --file $(LLVMIncludePath)/clang-c/BuildSystem.h --file $(LLVMIncludePath)/clang-c/CXCompilationDatabase.h --excludeFunctions clang_index_getClientEntity,clang_index_setClientEntity,clang_createTranslationUnitFromSourceFile,clang_parseTranslationUnit,clang_parseTranslationUnit2,clang_parseTranslationUnit2FullArgv,clang_reparseTranslationUnit,clang_codeCompleteAt,clang_indexSourceFile,clang_indexSourceFileFullArgv,clang_disposeSourceRangeList,clang_annotateTokens,clang_annotateTokens,clang_disposeTokens,clang_tokenize,clang_disposeStringSet,clang_getCompletionParent,clang_disposeCXPlatformAvailability,clang_getCursorPlatformAvailability",
-      "environmentVariables": {
-        "LLVMIncludePath": "C:\\Program Files\\LLVM\\include"
```

Comment:
These weren't needed as the actual setting was coming from the csproj file, which is probably better.